### ISSUE TITLE
Ban @GroupAnonymousBot from TELEGRAM_ADMIN

### ISSUE THREAD

**Issue by dgongut** [dgongut/docker-controller-bot#25]
<img width="1161" alt="image" src="https://github.com/dgongut/docker-controller-bot/assets/48691032/5943274f-9037-463c-abd7-80753b190463">

If spoken to from a group via an "invisible" administrator, the userID of the speaking user is not that of the REAL administrator. It is actually that of @GroupAnonymousBot. It can lead to confusion and someone can put the userid 1087968824 as TELEGRAM_ADMIN which is incredibly dangerous.

This would open the door for anyone to add your bot to a group and control your server.

**TLDR: DON'T PUT 1087968824 AS TELEGRAM_ADMIN**

**Comment by dgongut** [comment]
Solved in v2.3.3
